pytorch/serve
# added check for no objects detected in object detector

## Description

Fixed object detector default handler for "no objects detected" by the model.

Fixes #435 

## Type of change

## Feature/Issue validation/testing

 - Start torchserve
 - register fastrcnn model from model zoo
 - run inference with a [blank image](https://user-images.githubusercontent.com/26594595/84241556-3fbd6700-ab1d-11ea-9d6b-1e874f318c54.jpg)
 - The request should succeed with response "No object detected"

## Checklist:
